_data/projects/nunit.yml

```diff
@@ -2,10 +2,17 @@ name: NUnit
 desc: NUnit is the most popular and widely used unit test framework for .NET
 site: https://github.com/nunit/nunit
 tags:
-- ".NET"
+- NUnit
+- .NET
+- .NET Core
```

Comment:
Awesome! You should be good to go - project adds usually get approved pretty quick.